index.html

```diff
@@ -3,9 +3,23 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Hello Microverse!</h1>
-    <h3> Day 1 of Microverse </h3>
+    <h1 class="title">Hello Microverse Blog!</h1>
+    <div class="post">
+    <h1 class="post-title"> Day 1 </h1>
```

Comment:
- **[OPTIONAL]** For HTML & CSS practice there should be just one `h1` tag in the file we could use `h2` tag, please change it. 🙏🏽

Reply:
I have those condition as you said:
```Add a .gitignore file.
Add node_modules to gitignore file.
A test.md file with the text "This file should be ignored by git".```